### DISCUSSION TITLE
Whether the sidebar pops up should be optional

### DISCUSSION THREAD

**Discussion by mingyEx** [codewars/codewars.com#2905] [Ideas]
I think the left sidebar of codewars should have a function similar to `www.octotree.io`'s `Show sidebar on hover` , it should be able to choose whether to pin it, to prevent when my mouse accidentally moves to the left sidebar,
If I click the mouse, I will accidentally click on the link that pops up.

**Comment by JohanWiltink** [comment]
Yes _please._

( A tooltip on mouseover the pinned bar would be nice. )
